src/components/InversionSolutionMfdTab.tsx

```diff
@@ -75,7 +75,12 @@ const InversionSolutionMfdTab: React.FC<InversionSolutionMfdTabProps> = ({
     maxMagnitude = 9.0;
     minMagnitude = 5.0;
   }
-
+  //Removes filename from inversion data
+  const cleanedMeta = data?.node?.meta?.filter((el) => {
+    return el?.k !== 'rupture_set';
```

Comment:
@benjamineac can you also remove 'rupture_set_file_id' as it's not useful context here.

Reply:
Yep!